package.json

```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "jsonpath": "^0.2.2",
-    "lodash": "^4.16.3",
+    "lodash": "4.15.0",
```

Comment:
Out of curiosity, why down grade lodash here?

Reply:
Just to make sure it's consistent with our other projects.